### PR TITLE
[release-0.7] docs: mark release v0.7 as unsupported

### PR DESCRIPTION
**Pull request by marquiz**
Copied what #1132 did. Outcome will be similar (https://kubernetes-sigs.github.io/node-feature-discovery/v0.6)

### CODE REVIEW

**Comment by marquiz** [comment]
/retitle [release-0.7] docs: mark release v0.7 as unsupported

File: source/cpu/power_stub.go
Comment (PiotrProkop) [inline source/cpu/power_stub.go:1]: is this change intentional?

File: source/cpu/power_stub.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build !amd64
```
Comment (ArangoGutierrez) [inline source/cpu/power_stub.go:1]: what does this has to do with an html change?

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]

File: source/cpu/power_stub.go (hunk 1 above)
Comment (marquiz) [reply source/cpu/power_stub.go]: Nothing 🤨 Thanks for noting, fixed. I don't understand what gofmt magic I managed to do get this...
